static/css/base.css

```diff
@@ -455,7 +455,7 @@
 }
 
 @supports not (scrollbar-gutter: stable) {
-  .sl-scroll-lock {
+  .sl-scroll-lock body {
```

Comment:
What is this change for?

Reply:
It's the same situation as https://github.com/GoogleChrome/chromium-dashboard/pull/3639 -- starting the app changes this file -- but I didn't make a separate PR for it this time.